src/middlewared/middlewared/plugins/cloud/model.py

```diff
@@ -0,0 +1,64 @@
+from sqlalchemy.ext.declarative import declared_attr
+
+from middlewared.schema import Bool, Cron, Dict, Int, List, Str
+import middlewared.sqlalchemy as sa
+from middlewared.validators import Range, Time
+
+
+class CloudTaskModelMixin:
+    id = sa.Column(sa.Integer(), primary_key=True)
+    description = sa.Column(sa.String(150))
+
```

Comment:
There is a bunch of whitespace in this mixin. Can we remove it? Obviously the credential_id attribute can have spaces around it but not sure about the other seemingly random whitespace.